File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{

    float change;
    do
    {
        // Ask user for change
        change = get_float("How much change is owned: ");
    }
    while (change < 0);


    int convertCents = round(change * 100); //convert dollars to cents
    int coins = 0;

    while (convertCents >= 25) //loop for quaters
    {
        convertCents = convertCents - 25;
        coins++;
    }
    while (convertCents >= 10) //loop for dimes
    {
        convertCents = convertCents - 10;
        coins++;
    }
    while (convertCents >= 5) //loop for nicles
    {
        convertCents = convertCents - 5;
        coins++;
    }
    while (convertCents >= 1 && convertCents > 0) //loop for pennies
    {
        convertCents = convertCents - 1;
        coins++;
    }

    printf("%d\n", coins); //print the coins

}

File: pset6/cash/cash.py
# Import get_float from cs50
from cs50 import get_float

# Loop until user provides valid input
while True:
    cash = get_float("Change owed: ")

    if cash > 0:
        break

coins = 0

# Convert decimals to round numbers
cash = round(cash * 100)

# Loop until coins = 0 while incrementing coins on the way
while cash >= 25:
    cash -= 25
    coins += 1

while cash >= 10:
    cash -= 10
    coins += 1

while cash >= 5:
    cash -= 5
    coins += 1

while cash >= 1:
    cash -= 1
    coins += 1

print(coins)